### ISSUE TITLE
Setter by property: false positive warning

### ISSUE THREAD

**Issue by vepain** [best-doctor/flake8-class-attributes-order#24]
# False positive description

```python
class Foo:

  __ACLASSATTR = 0

  def __init__(n: int):
    self.__n = n

  @property
  def n(self):
    """The n getter."""
    return self.__n

  @n.setter
  def n(self, new_n: int):
    """The n setter befor @classmethod creates a warning."""
    self.__n = new_n

  @classmethod
  def a_classattr(cls):
    return cls.__ACLASSATTR
```

### What expected (to my mind)
`no warning`

### Current result
`Foo.n should be after Foo.a_classattr`

### Discussion

Because of the false positive result of _mypy_ (see https://github.com/python/mypy/issues/1465) the two warnings make the problem unfeasible.
But I think it is a great idea to keep getter and setter near to each other for the same attribute. 
So let consider you have to keep setter just after the getter

## Versions
- **Python:** `3.9.5`
- **flake8:** `3.9.0`
- **flake8-class-attributes-order:** `0.1.2`

---

Thank you for your work!

**Comment by aktentasche** [comment]
I second this bug, my minimum working example is:

```
class LogMessage:
    @property
    def package(self) -> str:
        return ""

    @package.setter
    def package(self, value: str) -> None:
        self.bar = value

    @property
    def message(self) -> str:
        return ""

    @message.setter
    def message(self, value: str) -> None:
        self.foo = value
```

When I group the setters the error goes away.

**Comment by Mogost** [comment]
Also faced this. Seems very reasonable to keep property and setter together.
